.github/workflows/dotnet.yml

```
@@ -4,27 +4,26 @@ on: [push, pull_request]
-    - name: Test
-      run: dotnet test --no-build --verbosity normal 
-        /p:CollectCoverage=true 
-        /p:CoverletOutput=../results/ 
-        /p:MergeWith=../results/coverage.info 
-        /p:CoverletOutputFormat=lcov 
-    - name: Publish coverage report to coveralls.io   
-      uses: coverallsapp/github-action@master   
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }} 
-        path-to-lcov: test/results/coverage.info 
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)